.github/workflows/tsc.yaml

```diff
@@ -0,0 +1,18 @@
+name: TSC
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
+jobs:
+  eslint:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Install tsc
+      run: yarn
+
+    - name: Run ESLint
```

Comment:
wrong name

Reply:
same on line 9. whats this for just a compile test?

Reply:
Yeah, just ensuring there are no TypeScript errors. I changed the name of the check to be more explicit.